<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #fffff0;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="checkBox_valuate_object">
     <property name="styleSheet">
      <string notr="true">QCheckBox {
    spacing: 8px;
    font-size: 14px;
    color: #333;
}

QCheckBox::indicator {
    width: 20px;
    height: 20px;
    border: 2px solid #0078d7;
    border-radius: 4px;
    background-color: white;
}

QCheckBox::indicator:hover {
    border: 2px solid #005fb8;
    background-color: #f0f8ff;
}

QCheckBox::indicator:checked {
    image: url(:/icons/checkbox_checked.svg);  /* или custom-галочка */
    background-color: #0078d7;
    border: 2px solid #0078d7;
}

QCheckBox::indicator:disabled {
    background-color: #eeeeee;
    border: 1px solid #bbbbbb;
}
QCheckBox::indicator:checked {
    background-color: #0078d7;
    border: 2px solid #0078d7;
}

</string>
     </property>
     <property name="text">
      <string>Оценка жилого дома на земельном участке</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true">QLabel {
    font-size: 14px;
    color: #333333;
    font-weight: bold;
    padding: 4px;
}
</string>
     </property>
     <property name="text">
      <string>Дата отчёта</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDateEdit" name="dateEdit">
     <property name="styleSheet">
      <string notr="true">QDateEdit {
    border: 1px solid #c0c0c0;
    border-radius: 6px;
    padding: 4px 8px;
    background-color: #ffffff;
    color: #333333;
    font-size: 14px;
}

QDateEdit:hover {
    border: 1px solid #0078d7;
}

QDateEdit::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 22px;
    border-left: 1px solid #c0c0c0;
    background-color: #f0f0f0;
}

QDateEdit::down-arrow {
    image: url(:/icons/calendar.svg);  /* можно указать PNG/ICO */
    width: 16px;
    height: 16px;
}
</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="OkNotButton">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLineEdit" name="NumberEdit">
     <property name="styleSheet">
      <string notr="true">
    border: 2px solid rgb(100, 100, 255);  /* цвет и толщина границы */
    border-radius: 10px;                  /* радиус скругления углов */

</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>OkNotButton</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OkNotButton</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
